index.js

```diff
@@ -0,0 +1,34 @@
+const myModule = require('./myModule.js');
+var 
+gm = require('gm');
+
+var favoriteFoods = myModule.favoriteFoods();
+
+for (let i = 0; i < favoriteFoods.length; i++) {
+        console.log(favoriteFoods[i]);
+    }
+
+// console.log the width and height of the image
+gm('./welcome2.png')
+.size(function (err, size) {
+    if (!err)
+    console.log(size.width);
+    console.log(size.height)
```

Comment:
Just a note that your if statement does not have curly braces, so it is not containing the two console.logs on lines 15 and 16.  Does that make sense?